hw1/Sem01Task06.py

```diff
@@ -0,0 +1,30 @@
+# Задача 6:
+# Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером.
+# Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна
+# сумме последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6.
+# Вам требуется написать программу, которая проверяет счастливость билета.
+
+def CountSumNumbers(inNum):  # функция подсчета суммы цифр в числе
+    sum = 0  # инициализируем переменную под сумму
+    while (inNum != 0):
+        sum = sum + inNum % 10  # прибавляем к сумме остаток от целочисленного делеения на 10
+        # "отбрасываем" от числа младший разряд, используя целочисленное делеение на 10
+        inNum = inNum // 10
+
+    return sum
+
+
+# спрашиваем у пользователя номер билета
+ticketNum = input("Введите 6-ти значный номер билета:")
+
+# проверяем ввел ли пользователь 6-ти значное число
+while not (str.isnumeric(ticketNum) and len(ticketNum) == 6):
```

Comment:
Не обязательно работать через метод класса, можно работать через метод объекта.
```suggestion
while not (ticketNum.isnumeric() and len(ticketNum) == 6):
```